Sources/KlaviyoSwift/KlaviyoState.swift

```diff
@@ -190,7 +193,7 @@ struct KlaviyoState: Equatable, Codable {
         }
         var attributes = profile.data.attributes
         var location = profile.data.attributes.location ?? .init()
-        var properties = profile.data.attributes.properties.value as? [String: Any] ?? [:]
+        let properties = profile.data.attributes.properties.value as? [String: Any] ?? [:]
```

Comment:
This is unrelated. I messed this up in the previous release so fixing it now.

Reply:
nice - I fixed this in a pr too.